swap_meet/vendor.py

```diff
@@ -1,2 +1,112 @@
+from operator import itemgetter
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+        
+    
+    def add(self, item):
+    
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        return False
+
+    def get_by_category(self, category):
+        items_list = []
+        for item in self.inventory:
+            if item.category == category:
+                items_list.append(item)
+        return items_list
+     
+
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        elif my_item in self.inventory and their_item in other_vendor.inventory:
```

Comment:
Seeing as though you already have an if statement to check if the items aren't in the inventory, this else if probably won't be needed.